app/scss/components/_buttons.scss

```diff
@@ -16,11 +16,10 @@
  */
 .button{
   display: inline-block;
-  padding: .8rem 1.5rem;
+  padding: .8rem 1.3rem;
   background: #888;
   color: $white;
   border: solid thin rgba($black, .1);
-  text-shadow: 0 -.05em .02em rgba($black, .5);
```

Comment:
Eyeballing it with and without text-shadow, it looks better without. I would also advocate for a flat design, I think it fits better with the modern look of the website.